src/main/scripts/src/app/workspace/organization/organization-settings.component.ts

```diff
@@ -17,10 +17,106 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component} from '@angular/core';
+import {Component, OnInit, Output, EventEmitter, ViewChild, ElementRef} from '@angular/core';
+import {ActivatedRoute, ParamMap, Router} from '@angular/router';
+
+import {Organization} from '../../core/dto/organization';
+import {WorkspaceService} from '../../core/workspace.service';
+import {OrganizationService} from '../../core/rest/organization.service';
+import {HttpResponse} from '@angular/common/http';
+import {ProjectService} from '../../core/rest/project.service';
+import {Project} from "../../core/dto/project";
 
 @Component({
-  templateUrl: './organization-settings.component.html'
+  templateUrl: './organization-settings.component.html',
+  styleUrls: ['./organization-settings.component.scss']
 })
-export class OrganizationSettingsComponent {
+export class OrganizationSettingsComponent implements OnInit {
+
+  private organization: Organization;
+  private organizationCode: string;
+  private errorMessage: any;
```

Comment:
Useless variable.. we have `notificationService` now.

Every: `this.errorMessage = error` -> `this.notificationService.error('Error', 'Error simplified for user');`